jupyterhub/tasks/main.yml

```diff
@@ -321,7 +310,7 @@
   when: ansible_os_family == 'RedHat'
 
 - name: Ensure hostname is in /etc/hosts file
-  lineinfile: dest=/etc/hosts regexp="{{ansible_hostname}}" line="{{ip}}  {{ansible_hostname}}"
+  lineinfile: dest=/etc/hosts regexp="{{ansible_hostname}}" line="{{ip}}" line="{{ansible_hostname}}"
```

Comment:
I don't think you can have multiple `line` parameters, can you? And even if you can, the IP address and hostname need to be on the same line, so it's not clear what you mean to fix here. @gsteffen1921 can you explain?

Reply:
You can't have multiple line parameters.  I've changed it back.  I was experimenting with the formatting of the lineinfile: and failed to return it back to the original format.
Wasn't sure if... 
lineinfile:
  dest: /etc/hosts 
  regexp: "{{ansible_hostname}}" 
  line:"{{ip}}  {{ansible_hostname}}"
would have been a better way to format.